src/index.njk

```diff
@@ -1,34 +1,81 @@
+      <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds govuk-!-mt-r6 govuk-!-mb-r6">
+        <h2 class="govuk-heading-l">Get started</h2>
+        <p class="govuk-body govuk-!-mb-r0">
+          Quickly build prototypes using the styles, components and patterns in this design&nbsp;system. Read a guide on <a href="/get-started/" class="govuk-link">how to get started</a>.
+        </p>
+
       </div>
       <div class="govuk-o-grid__item govuk-o-grid__item--full">
         {% include "../views/partials/_divider.njk" %}
       </div>
-      <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds govuk-!-mb-r4">
-        <h2 class="govuk-heading-l govuk-!-mt-r6">Get in touch</h2>
-        <p class="govuk-body govuk-!-mb-r0">If you’ve got a question, idea or suggestion share it in <a class="govuk-link" href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system">#govuk-design-system</a>, on cross-government Slack (<a class="govuk-link" href="slack://channel?team=T04V6EBTR&id=C6DMEH5R6">open in app</a>) or <a class="govuk-link" href="mailto:govuk-design-system-support@digital.cabinet-office.gov.uk">email the Design System team</a></p>
+      <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds govuk-!-mt-r6 govuk-!-mb-r6">
+        <h2 id="get-in-touch" class="govuk-heading-l">Support</h2>
+        <p class="govuk-body">
+          The GOV.UK Design System is maintained by the Government Digital Service.
+          If you’ve got a question, idea or suggestion you can:
+        </p>
+        <ul class="govuk-list govuk-list--bullet govuk-!-mb-r0">
+        <li>
+          <a href="mailto:govuk-design-system-support@digital.cabinet-office.gov.uk" class="govuk-link">
+            email the Design System team
+          </a>
+        </li>
+        <li>
+          <a href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system" class="govuk-link">
+            get in touch on Slack</a> (<a class="govuk-link" href="slack://channel?team=T04V6EBTR&amp;id=C6DMEH5R6">open in app</a>)
+        </li>
+        </ul>
+      </div>
+      <div class="govuk-o-grid__item govuk-o-grid__item--full">
+        {% include "../views/partials/_divider.njk" %}
+      </div>
+      <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds govuk-!-mt-r6 govuk-!-mb-r6">
+        <h2 class="govuk-heading-l">Contribute</h2>
+
+
+        <p class="govuk-body">
+          To propose a new style, component or pattern for the GOV.UK Design System <a href="https://github.com/alphagov/govuk-design-system-backlog/blob/master/docs/CONTRIBUTING.md" class="govuk-link">read the contribution guidelines on GitHub</a>.
+        </p>
+
+        <p class="govuk-body govuk-!-mb-r0">
+          You can see what’s currently being worked on in the <a href="https://github.com/alphagov/govuk-design-system-backlog" class="govuk-link">GOV.UK Design System Backlog</a>.
+        </p>
+
+        <!-- <p class="govuk-body">
```

Comment:
Is this intentionally included? I'd prefer to just delete it if it's not 'ready' yet.